mobile/src/main/java/ch/epfl/sdp/mobile/state/StatefulVisitedProfileScreen.kt

```diff
@@ -115,3 +117,10 @@ private object EmptyProfile : Profile {
   override val followed: Boolean = false
   override val solvedPuzzles = emptyList<PuzzleId>()
 }
+
+data class VisitedProfileActions(
```

Comment:
Missing some documentation for the parameters / properties.

Reply:
Documented in [64a6544](https://github.com/epfl-SDP/android/pull/303/commits/64a6544a2aa602d5d620d6672eaab107e301e665)